src/test/java/org/fogbowcloud/manager/core/ApplicationFacadeTest.java

```diff
@@ -12,1701 +12,2209 @@
-			this.application.deleteCompute(order.getId(), federationTokenValue);
-			Assert.fail();
-		} catch (UnauthenticatedUserException e) {
-			Assert.assertEquals(OrderState.OPEN, order.getOrderState());
-		}
-	}
+        try {
+            // exercise
+            this.application.deleteCompute(order.getId(), FEDERATION_TOKEN_VALUE);
+            Assert.fail();
+        } catch (UnauthenticatedUserException e) {
+            // verify
+            Assert.assertEquals(OrderState.OPEN, order.getOrderState());
+        }
+    }
 
-	@Test
-	public void testDeleteComputeOrderWithFederationUserUnauthenticated() throws Exception {
-		Mockito.doNothing().when(this.aaaController).authenticate(Mockito.anyString());
+    // test case: When try calling the deleteCompute() method with a Federation User without
+    // authentication, it must throw UnauthenticatedUserException and the Order remains in the same
+    // state.
+    @Test
+    public void testDeleteComputeOrderWithFederationUserUnauthenticated() throws Exception {
 
-		Order order = createComputeOrder();
-		Mockito.doThrow(new UnauthenticatedUserException()).when(this.aaaController).getFederationUser(Mockito.anyString());
+        // set up
+        Order order = createComputeOrder();
+        OrderStateTransitioner.activateOrder(order);
 
-		OrderStateTransitioner.activateOrder(order);
+        Mockito.doNothing().when(this.aaaController).authenticate(Mockito.anyString());
 
-		Mockito.doNothing().when(this.aaaController).authorize(Mockito.any(FederationUser.class),
-				Mockito.any(Operation.class), Mockito.any(Order.class));
+        Mockito.doNothing().when(this.aaaController).authorize(Mockito.any(FederationUser.class),
+                Mockito.any(Operation.class), Mockito.any(Order.class));
 
-		try {
-			String federationTokenValue = "";
-			this.application.deleteCompute(order.getId(), federationTokenValue);
-			Assert.fail();
-		} catch (UnauthenticatedUserException e) {
-			Assert.assertEquals(OrderState.OPEN, order.getOrderState());
-		}
-	}
+        Mockito.doThrow(new UnauthenticatedUserException()).when(this.aaaController)
+                .getFederationUser(Mockito.anyString());
 
-	@Test
-	public void testDeleteComputeOrderUnauthorizedOperation() throws Exception {
-		Mockito.doNothing().when(this.aaaController).authenticate(Mockito.anyString());
+        try {
+            // exercise
+            this.application.deleteCompute(order.getId(), FEDERATION_TOKEN_VALUE);
+            Assert.fail();
+        } catch (UnauthenticatedUserException e) {
+            // verify
+            Assert.assertEquals(OrderState.OPEN, order.getOrderState());
+        }
+    }
 
-		Order order = createComputeOrder();
-		OrderStateTransitioner.activateOrder(order);
+    // test case: When try calling the deleteCompute() method performing an operation without
+    // authorization, it must throw UnauthorizedRequestException and the Order remains in the same
+    // state.
```

Comment:
improve the description of the test case to reflect what the test do (or change the test to address the description). see my previous comments

Reply:
Fixed the name and description of the test case.